packages/models-library/tests/test_service_settings_labels.py

```diff
@@ -387,3 +392,189 @@ def test_not_allowed_in_both_permit_list_and_outgoing_internet():
         f"Not allowed common_containers={{'{container_name}'}} detected"
         in f"{exec_info.value}"
     )
+
+
+@pytest.fixture
+def vendor_environments() -> dict[str, Any]:
+    return {
+        # NO list or dict!
```

Comment:
not sure I get this comment. what does that mean? especially since there is a list right after...

Reply:
Sorry! Forgot to update this comment!